src/types/Theme.js

```diff
@@ -229,6 +229,9 @@ export type Theme = {
   tabBarIcon: string,
   tabBarIconHighlighted: string,
 
+  extraTabBarIconFont: string,
+  extraTabBarIconName: string,
+
```

Comment:
Please put these in the `extraTab` section of the app config. These icon properties do not need to change dynamically.

We have agreed that the theme is for colors, not things like dimensions or icons or such - otherwise the whole UI would be in the theme.

Reply:
Funny you say that, I actually started with everything in the config file. However I felt that icons were more of a theme concept and the theme actually does contain icons and has for quite some time. This allows for different icons in light vs dark mode. Granted this specific icon isn't a PNG so its color can be changed without specifying a different icon, but for consistency, it felt correct to put it into the theme like the other icons. Also, a dark vs light theme might want a different vector icon for thinner/thicker lines that might contrast better with different colors. I'll admit, that's a bit of a stretch but still a valid thought

I can move the `name` part out to the config as that admittedly should never need a difference between light and dark, but it does make make the two values a bit disconnected.

Reply:
Right, the only reason we have PNG's in there is because PNG's contain color information. However, this is a bad thing. If we could get away from colored PNG's, that would be preferable to stuffing images in the theme.

Of course, the argument about line width would apply to anything. We could put literally every dimension in our app into the theme, and then have the file be 10,000 lines long and unmaintainable for "maximum flexibility". This is obviously this is a bad idea. Therefore, we need a *strong* bias in favor of putting as little stuff as possible into our theme. The default position should be to keep stuff out of the theme, and then only add them when proven absolutely necessary.